app/models/user.rb

```diff
@@ -12,6 +14,19 @@ def name
     @name ||= self[:name].presence || email.split("@").first
   end
 
+  def self.from_omniauth(access_token)
+    data = access_token.info
+    user = User.where(email: data['email']).first
```

Comment:
ℹ️ `where` + `first` => `find_by`

But usually, the `from_omniauth` class method uses the `first_or_create`